ethereum/rpc/namespaces/eth/api.go

```diff
@@ -991,9 +983,9 @@ func (e *PublicAPI) GetProof(address common.Address, storageKeys []string, block
 	}, nil
 }
 
-// setTxDefaults populates tx message with default values in case they are not
+// SetTxDefaults populates tx message with default values in case they are not
 // provided on the args
-func (e *PublicAPI) setTxDefaults(args rpctypes.SendTxArgs) (rpctypes.SendTxArgs, error) {
+func (e *PublicAPI) SetTxDefaults(args rpctypes.SendTxArgs) (rpctypes.SendTxArgs, error) {
```

Comment:
I think we can keep it private

Reply:
Yes, this is an old change used when we were trying this function on another file. I'll revert this change.